contentgrid-spring-boot-actuators/src/main/java/com/contentgrid/spring/boot/actuator/SystemProperties.java

```diff
@@ -0,0 +1,11 @@
+package com.contentgrid.spring.boot.actuator;
+
+import java.util.Map;
+import lombok.Data;
+
+@Data
+public class SystemProperties {
```

Comment:
This is a confusing name, because "system" properties also means the `system.deployment.id` properties etc.